extensions/azurecore/extension.webpack.config.js

```diff
@@ -13,5 +13,8 @@ module.exports = withDefaults({
 	context: __dirname,
 	entry: {
 		extension: './src/extension.ts'
+	},
+	externals: {
+		'keytar': 'commonjs keytar'
```

Comment:
why are we excluding this from the packaging?

Reply:
Keytar is an external and needs to be done like this or there will be build errors.

Reply:
Specifically the issue here is that keytar contains non-javascript components and as such can't be webpacked.